src/Core/Module/ModuleRepository.php

```diff
@@ -131,6 +131,20 @@ public function getUpgradableModules(): ModuleCollection
      * @return Module
      */
     public function getModule(string $moduleName): ModuleInterface
+    {
+        $coreModule = $this->getCoreModule($moduleName);
+
+        return $this->enrichModuleAttributesFromHook($coreModule);
+    }
+
+    /**
+     * @param string $moduleName
+     *
+     * @return Module
+     *
+     * Internal function used for internal module description and cache management
+     */
+    private function getCoreModule(string $moduleName): ModuleInterface
```

Comment:
```suggestion
    protected function getCoreModule(string $moduleName): ModuleInterface
```

As the method `getModule` can be overriden, we should let developer can use original code for a better developer experience.

Reply:
By default I set everything to private will there is no explicit needs but as I don't know needs, this habit may induce some errors ^^'

Thanks for advice ^^